hw_37/src/Tabs.tsx

```diff
@@ -0,0 +1,35 @@
+import './css/tabs.css';
+
+export type TabsProps = {
+    title: string;
+    active?: boolean | undefined;
+    disabled?: boolean
+};
+
+export const Tabs = ({ tabsArray }: { tabsArray: TabsProps[] }) => {
+    if (!tabsArray.length) {
+        return null;
+    }
+    return (<ul className="tabs">
+        {tabsArray.reduce((acc: any[] = [], arr) => {
```

Comment:
here length of tabsArray and the number of results elements are the same
prefer to use map, it's easy to read

Reply:
and avoid to use any

Reply:
changed reduce method to map